app/views/c_rater/argumentation_block/_runtime.html.haml

```diff
@@ -0,0 +1,87 @@
+- embeddables = arg_block_embeddables(@page, @run)
+- if embeddables.any?
+  .embeddables.arg-block
+    - any_feedback = embeddables.any? { |e| f = e.get_saved_feedback; f && f.feedback_text  }
+    .ab-feedback-header{:style => any_feedback ? '' : 'display: none'}
+      %h3 Feedback available!
+    - embeddables.each do |m|
+      .question
+        = render :partial => "#{m.class.name.underscore.pluralize}/lightweight", :locals => {:embeddable => m}
+        - fb = m.get_saved_feedback
+        - fb_visible = fb && !fb.feedback_text.blank?
+        - fb_dirty = fb && fb.outdated?
+        - fb_text = fb && fb.feedback_text
+        - fb_score = fb && fb.score
+        .ab-feedback{:id => "feedback_on_answer_#{m.id}",
+                     :style => fb_visible ? '' : 'display: none',
+                     :class => "ab-score#{fb_score}",
+                     'data-dirty' => fb_dirty ? 'true' : 'false'}
+          .ab-dirty{:style => fb_dirty ? '' : 'display: none'}
+            Answer has been changed, please resubmit to update feedback!
```

Comment:
The user visible strings in this file should be moved into config/locales/en.yml